src/parquet/encodings/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright 2015 Cloudera Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Headers: encodings
+install(FILES
```

Comment:
Ok parquet.h currently relies on rle-encoding.h which pulls in all them. Let's address this later but we should minimize exposing the internal components. I think we'll want to batch some of the current APIs anyway, and then we can move the implementation out of parquet.h